src/libfmt_macros/lib.rs

```diff
@@ -154,6 +154,7 @@ pub struct Parser<'a> {
     style: Option<usize>,
     /// How many newlines have been seen in the string so far, to adjust the error spans
     seen_newlines: usize,
+    pub arg_places: Vec<(usize, usize)>,
```

Comment:
so these are offsets into the format string? A comment would be nice (I'm using this struct in clippy ^^)

Reply:
Added docs.